datasets.py

```diff
@@ -73,3 +73,15 @@ def from_text(cls, text: str) -> Self:
             dp = DataPoint(row_values[0], row_values[1], row_values[2])
             data.append(dp)
         return cls(type_, accepted, reference, ionic_liquid, details, data)
+
+    @classmethod
+    def from_file(cls, file_path: str) -> list[Self]:
+        """Convert a file containing multiple datasets into a list of `Dataset`
+        instances.
+        """
+        with open(file_path, "r") as file:
+            file_data = file.read().split("\n\n")
```

Comment:
I think there may be some data files with multiple empty lines between datasets. This assumes there is only one. It's OK, since it passes test. However, try to take into account that the number of lines separating sets may vary.

In a moment, I will modify to cover that issue...

Reply:
I had to add a check in line 84, because the last string ended with '\n' after changes in the test.